### PR TITLE
Welcoming contact field

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by vasilvestre** [comment]
Hey, could you rebase your branch please ? Code of conduct should be in.
Where is Contact.md used please ?

File: CONTACT.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Welcome to the community.
+
+Settle differences by offering and accepting constructive critique, within desired privacy.
+Finding yourselves unable, e-mail Valentin Silvestre (the project maintainer) at vsilvestre.pro@gmail.com
```
Comment (artandor) [inline CONTACT.md:4]: Please add my adress to this, aswell, Nicolas Mylle, nicolas.mylle.pro@gmail.com

**Comment by comradekingu** [comment]
@vasilvestre Weblate, Remmina, and so on use CONTACT.md to great effect.
It saddens me when projects impose CoCs. I am not saying you can't have one, but I am here to contribute to libre software because it is libre, not anything that isn't.

**Comment by artandor** [comment]
> @vasilvestre Weblate, Remmina, and so on use CONTACT.md to great effect. It saddens me when projects impose CoCs. I am not saying you can't have one, but I am here to contribute to libre software because it is libre, not anything that isn't. I ask you not to use my CONTACT.md if you want to have one.

The code of conduct above only restrict potential negative behaviors or disrespectfull ones. I don't think your freedom is affected in any way except the way you can treat people in here.
It might help us protect the project or the people working on it at some point ; so, for my part, I am for keeping it, it costs nothing and helps people understand the kind of projects they are contributing to.

**Comment by comradekingu** [comment]
@artandor That is not the case.
It starts singling out people for immutable characteristics. Often aspects beyond their control.
That is not warranted, and it isn't a positive psychological tool. It is the opposite of cognitive behavioral therapy, which is the strongest tool of clinical psychology.

It restricts community to the corporate ideals of professionalism, strips it of sexuality, censors language, and it is a political document that bans politics.
Total excommunication is encouraged, with retroactive censorship. It moves decisions away from project maintainers over to "community leaders", and it keeps itself above all.

For anything to be disrespectful, negative behavior, or "harassment", an objective reality must be agreed on. What this text-file does is replace that with experience, so as to totally forego intent.
That is an ungovernable fallacy. It sounds nice, but for any practical application, it does nothing other than inject the rank authoritarianism above.
There are a great many cases of disasters that could only be a result of it, and not a single thing good thing one couldn't do without one.

What it does is give anti-social people a list of things that are above board, and a structure to exploit by gaining its powers without vested interest.
It doesn't help the project, because donations (just experience) and interest (people leaving or not egaging) go down, and the level of conflict goes up (disasters and unproductive discussions).
Not only because it is unpopular, but because it is designed to escalate conflict beyond the control of individuals involved. It then requires secrecy as a departure from legal functioning.

Anti-social people, or their behavior is in no way governed by a text-file, any more than it is a good alternative to actual law governing harassment. You are then left with something weak individuals believe in, and that false confidence is dangerous. This starts as early as actually reading the document in good will.

**Comment by vasilvestre** [comment]
@comradekingu We may agree on having a Code of conduct that's less formal but I want to keep the informative message in the CODE_OF_CONDUCT file.
It's not a political issue here, GIthub have an integration for the CoC & I want to keep it. See here : 

image

**Comment by comradekingu** [comment]
@vasilvestre My message/text isn't less formal, it is less authoritarian.
I don't want to pose as if it is a code of conduct, because I don't write those.
It is neither a manifesto, decree, fatwa nor advert, but a means of contact is a possible feature in all of those.

Not that it takes away from its endless bloat and feature-creep, but the CoC used here was actually written by a GitHub employee, while at GitHub. That employee was later fired from GitHub for not being welcoming to newcomers.
A starker reminder that none of its good functions are also possible without one isn't possible.
It magically doesn't make anyone behave morally, or do what their job-description is, even if paid what I imagine to be industry tariffs.
There is no shortage of absolute disasters that could only have happened with it though.
GitHub itself doesn't use it, and it is redundant to one for GitHub that nobody seems to read.)
The field you are talking about is only there because clicking it is so easy, as evident from the most commonly used CoC being many versions out of date.

I don't believe for a second that the missing component for anyone was to display it more prominently, or for that matter to have more cookie-cutter solutions showing green lights to implement it.
I have myself been tasked with telling people what the rules are at the door. Who says it matters, and nobody remembers long text, and especially the undermining politics, because it is never a substitute for culture or law.
Nothing is done well by having unwritten rules in a text-file.
We are talking about a political document, as stated by its creator. That is the intention, and that was also the case with the catholic church, and the mafia.
Unfortunately their methods have made their way into my ecosystem, but one thing we can agree on is it would not be possible to put it in a libre license.
That didn't stop the author of it from trying, and there is now even a non-free licensing scheme made by this person.

Removing this CoC and replacing it with the specific CONTACT.md, or some remembered variant works. I will forever keep asking why anyone should go down the road of having a CoC, and I will keep an open mind, but thus far there are zero good reasons beyond "green light" and "important". The more openly communicated the removal, the better. Code of Conduct people and politics are the least popular in this ecosystem.

TL;DR It isn't good in terms of content, effect, or history, and it isn't good in terms of name or functionality either.
Nobody is ever asked if they want to have one, as even its most central underpinning is a lie.
People see it, and leave without donating or contributing.

**Comment by artandor** [comment]
After a lot of discussions between us maintainers and other Open Source communities, we've decided to refuse this pull request.

Thanks a lot for your participation and your insights. We hope you'll continue to contribute here, even if, considering what you said above, it is unlikely.

Have a nice day

**Comment by comradekingu** [comment]
Somehow backroom deals and discussions behind closed doors go hand in hand with not arguing ones case.
"Open" "source" and "community" are all aspects that suffer as a result.
That is how it gets to be this way. Nobody is asked when it is implemented, and everyone is thought to comply when it is there.
If translators want to contribute on Weblate, they aren't even made aware.

Replacing meritocracy with authoritarianism has nothing to do with community.

>We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.

Practically, it isn't even worth that wrapping.

As an aside from what people decide to do as maintainers, the only thing the document really does is to create an attack-vector
for people without vested interest to rule over a community, with powers surpassing that of maintainers. That isn't how anything got built, it isn't what people want to contribute to, and it only does anything distinct as facilitating abuse their non-earnt power.
That is all of its history, with no evidence to the contrary. How this dangerous bloatware came to waste so many man-hours is its only legacy.
